polls/tool/src/java/org/sakaiproject/poll/tool/producers/ResultsProducer.java

```diff
@@ -135,9 +135,11 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams,
 		
 		//get the number of votes
 		int voters = pollVoteManager.getDisctinctVotersForPoll(poll);
-		//Object[] args = new Object[] { Integer.valueOf(voters).toString()};
-		if (poll.getMaxOptions()>1)
-			UIOutput.make(tofill,"poll-size",messageLocator.getMessage("results_poll_size",Integer.valueOf(voters).toString()));
+		int totalVoters = externalLogic.getNumberUsersCanVote();
+		float percentVoters = (float) voters / (float) totalVoters;
```

Comment:
I suggest using BigDecimal instead of float to do this operation:
https://docs.oracle.com/javase/7/docs/api/java/math/BigDecimal.html
https://www.geeksforgeeks.org/bigdecimal-class-java/